.github/workflows/ci.yml

```diff
@@ -29,7 +32,9 @@ jobs:
       with:
         name: linux-binaries
         path: |
-          dist/jitsi-meet-amd64.deb
+          dist/*.deb
+          dist/*.changes
+          dist/*.buildinfo
           dist/jitsi-meet-x86_64.AppImage
```

Comment:
So these will be uploaded as artifacts and we can use the links from the release to push it ... then we can trigger a remote ci job via web request, I think - the link and credentials will come from the env variables ... 
Can we trigger the jenkins job once the artifacts are uploaded and publicly available?

Reply:
I think that should be doable from the ci.yml file yeah.